src/parser/Parser/Parser.hs

```diff
@@ -900,73 +917,89 @@ expression = makeExprParser expr opTable
                   op "/=" DIV_EQUALS]
                 ]
 
+      opRange emeta = do
```

Comment:
the name `opRange` may seem to suggest that it has something to do with `Ranges` in Encore. As I understood it, this operation is used whenever we have an expression in which there may be ambiguity about the pos (column) of the error, e.g. `x ! foo()`, the errors occurs in the `x` or in the `foo`? I don't know yet what better name we can give it but, if the name doesn't change, I think that a comment can help in understanding its _raison d'être_